src/components/ImagePicker/ImagePicker.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useMemo } from 'react';
+import { View, StyleSheet, Alert } from 'react-native';
+import { AssetsSelector } from 'expo-images-picker';
+import { Ionicons } from '@expo/vector-icons';
+import { Asset, MediaType } from 'expo-media-library';
+
+// IOS users , make sure u can use the images uri to upload , if your getting invalid file path or u cant work with asset-library://
+// Use = > getImageMetaData: true which will be little slower but give u also the absolute path of the Asset. just console loge the result to see the localUri
+
+// See => https://docs.expo.dev/versions/latest/sdk/media-library/#assetinfo
+
+const ImagePicker = ({ navigation, route }: any) => {
+  const onSuccess = (data: Asset[]) => {
+    navigation.navigate({
+      name: 'Camera',
+      params: { uris: data.map(asset => asset.uri) },
+      merge: true,
+    });
+  };
+  const styles = StyleSheet.create({
```

Comment:
Move this to styled component in separate file + add a reasonable margin so that iOS devices work